tools/run_tests/run_interop_tests.py

```diff
@@ -1019,7 +1062,9 @@ def aggregate_http2_results(stdout):
       _LANGUAGES_FOR_HTTP2_BADSERVER_TESTS, resultset, num_failures,
```

Comment:
You need to add these new cases to "http2_badserver_test_cases, _LANGUAGES_FOR_HTTP2_BADSERVER_TESTS" in order for them to show up in the HTML report. 
I suggest renaming these vars by removing "bad" to incorporate both cases. 
You'll also need to modify https://github.com/grpc/grpc/blob/master/tools/run_tests/python_utils/report_utils.py#L83 and https://github.com/grpc/grpc/blob/master/tools/run_tests/interop/interop_html_report.template#L109 accordingly. 
It'll be nice to put them in the same table and call it "HTTP/2 Server Tests" as in https://grpc-testing.appspot.com/job/gRPC_interop_master/HTML_Report/. If neg and pos cases work with different client languages, you can pass a superset of those. I believe if the language is not supported, it'll just say "unimplemented".